src/main/modules/nunjucks/index.ts

```diff
@@ -40,6 +40,10 @@ export class Nunjucks {
       return typeof prop === 'function' ? prop(this.ctx) : prop;
     });
 
+    nunEnv.addGlobal('getContentSafe', function (prop: any | string): nunjucks.runtime.SafeString {       
```

Comment:
What is this doing?

Reply:
the global config for we set for nunjucks is to escape content. The getcontent will render anything passed to it as escaped. There are some cases where we need to pass text with html (e.g. to apply bold styles for part of the text). This is why getcontentsafe was needed. I initially tried to modify getcontent but this introduced some undefined errors for other content being processed.  Discussed with Taut and decided to just apply it to field values and text.

Reply:
Is it possible to create some test cases around this?